src/main.js

```diff
@@ -144,8 +144,7 @@ module.exports = function () {
         desc: 'Launches github PR UI for the specified topic branch',
         entryPoint: lazyRequire('./create-pr'),
         noChdir: true
-    },
-    {
+    }, {
```

Comment:
Just a minor nit that I would have fixed this one in a separate commit.

Reply:
if it was more than 2, maybe 3 I would. Otherwise not worth the time.